clients/devtools-entry.js

```diff
@@ -7,7 +7,7 @@
 
 const lighthouse = require('../lighthouse-core/index.js');
 const RawProtocol = require('../lighthouse-core/gather/connections/raw.js');
-const log = require('lighthouse-logger');
```

Comment:
if we're doing this it seems like we might as well just directly require in from `../lighthouse-logger` and expose the logger object for setting level and whatnot at `require('lighthouse')`, then we wouldn't need to worry about this publish business.

honestly, the only reason it was separate package at all was so that ChromeLauncher could share the logging functions/levels but I'm thinking there are much better ways to go about sharing a log object with chrome launcher than to continue to jump through all these hoops. I realize this is a bit more than you probably wanted to chew on for this change 😆  but if we're rewriting all the logger require statements to make this work, it's worth discussing.

Reply:
What's the better way to share this code with ChromeLauncher? I would very much like to pull this package back into core.

Reply:
I was thinking ChromeLauncher could just accept a `logger` parameter as an option that conforms to the API we have and we can pass `lighthouse-logger` into, but we could use regular ol' `debug` by default